dockers/docker-sflow/port_index_mapper.py

```diff
@@ -45,7 +45,15 @@ def update_db(self, ifname, op):
         index = port_util.get_index_from_str(ifname)
         if op == 'SET' and index is None:
             return
-        ifindex = if_nametoindex(ifname)
+
+        # catch system error and log as warning level instead of
+        # error level in case interface was already deleted
+        ifindex = None
+        try:
+            ifindex = if_nametoindex(ifname)
+        except OSError as e:
+            logger.log_warning("%s" % str(e))
```

Comment:
Dont we need to return upon OSError exception?

Reply:
to clean from cur_interfaces and DB did not return upon handling OSError.